app/models/user.rb

```diff
@@ -41,8 +42,8 @@ class User < ActiveRecord::Base
   before_save :set_token
   after_destroy :destroy_drupal_user
 
-  def self.search(query)
```

Comment:
Ah, but perhaps we want to keep the more generalized `.search()` too, for searching by both username and bio -- for situations where we want both...